config/routes.rb

```diff
@@ -6,6 +6,10 @@
   namespace :api do
     namespace :v1 do
       resources :books, only: %I[index show]
+
+      scope 'users/:user_id' do
```

Comment:
you can use here the resources of users instead i think  🤔

Reply:
but it isn't a resource of the user...so .... what I do?